AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1029,6 +1029,7 @@ hello7app.com##.email_box
 gazeta.ru##.telegram_info
 winaero.com#?#aside:has(> h2:contains(Subscribe))
 russiancouncil.ru##.form_subscribe
+theblockcrypto.com##.fixed-bottom
```

Comment:
I would suggest: theblockcrypto.com##body > .fixed-bottom
Also please move the rule to /popups.txt